### DISCUSSION TITLE
Optional Vim motions in the Godot built-in code editor

### DISCUSSION THREAD

**Discussion by kubgus** [godotengine/godot-proposals#11145] [Editor]
**Problem:**
No way to enable Vim motions in the built-in editor. (Note: I know it's possible to use external editors with Godot, but most people don't go through that hassle)

**Solution:**
Add an option to enable Vim motions to the editor settings.

**Benefits:**
- More effective code editing
- It's unique, other standard game engine editors don't have this feature; it might open the engine to a new market
- It's gonna be a cool Twitter post

*I want to make a PR myself but I'm currently unable to allocate time for it so I'm putting this idea out there for now.*

**Comment by Calinou** [comment]
This was proposed in the past and was rejected: https://github.com/godotengine/godot-proposals/issues/622

> - It's gonna be a cool Twitter post

I'm afraid this is not how features are considered for inclusion in Godot :slightly_smiling_face: 

See [Best practices for engine contributors](https://docs.godotengine.org/en/stable/community/contributing/best_practices_for_engine_contributors.html).

**Comment by passivestar** [comment]
This is one of the reasons godot needs [global addons](https://github.com/godotengine/godot-proposals/issues/831)
